IBAnimatable/GradientType.swift

```diff
@@ -84,7 +87,7 @@ public enum GradientType: String, IBEnum {
   case redMist
   case tealLove
   case neonLife
-  case manOfSteel
+  case manofSteel
```

Comment:
You're right. We could use `capitalized` and change the first character later after removing all spaces and e.t.c?